.github/workflows/publish-dfxvm-install-script.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish dfxvm install script
+
+on:
+  push:
+    branches:
+      - sdk-1278-dfxvm-install-script
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  # When getting Rust dependencies, retry on network error:
+  CARGO_NET_RETRY: 10
+  # Use the local .curlrc
+  CURL_HOME: .
+
+jobs:
+  publish-manifest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install shfmt
+        run: go install mvdan.cc/sh/v3/cmd/shfmt@latest
+      - name: Generate
+        run: |
+          shellcheck --shell=sh public/install/*.sh --exclude SC2154,SC2034,SC3003,SC3014,SC3043
+          ~/go/bin/shfmt -d -p -i 4 -ci -bn -s public/install/*.sh
+          sed -i "s/@revision@/${GITHUB_SHA}/" public/install/999_footer.sh
+          mkdir _out
+          cat public/install/*.sh > _out/install.sh
+          sed -i "
```

Comment:
I think adding short explanation of what is going on here would be beneficial

Reply:
With respect to this PR, that is a request for adjacent work.

This PR copies https://github.com/dfinity/sdk/blob/master/.github/workflows/publish-manifest.yml and makes three small changes: 
1) run the workflow when changes are pushed to a different branch
2) push the build install script to a different branch
3) don't add public-manifest.yml to the branch

The public-manifest.yml script was in turn copied here in [PR-2198](https://github.com/dfinity/sdk/pull/2198) from some other process.  I believe it was a nix process, and as I recall, there were no beneficial short explanations there either.